app/views/markets/show.html.erb

```diff
@@ -1,25 +1,23 @@
-<div class="container" style="height:  width:60%" >
+<div class="container">
   <h1><%= @market.name %></h1>
   <p><%= @market.category.name %></p>
   <p><%= @market.address%></p>
   <p><%= @market.description%></p>
-
-</div>
-
-<div style="width: 100%; height: 600px;">
-  data-controller="map"
-  data-map-markers-value="<%= @marker.to_json %>"
-  data-map-api-key-value="<%= ENV['MAPBOX_API_KEY'] %>"</div>
-
```

Comment:
the problem here was that `style` is not the only attribute of `div` that you want here.

`data-controller="map"` is also an attribute of this div that is important to mapbox to see so that it knows to replace this dig with content

`data-map-markers-value="<%= @marker.to_json %>"` is also an attribute of this `div`, as is the next line. These are attributes just like `class="something"` or `href="www.url.com/img.gif`. But here, the `div` got closed after the `style` attribute was set, so these other attributes were just floating in the code like a fart in space